src/status_im/utils/prices.cljs

```diff
@@ -40,6 +41,11 @@
                                      :last-day (:OPEN24HOUR entry)}}))}))))
 
 (defn get-prices [from to mainnet? on-success on-error chaos-mode?]
+  (log/debug "[prices] get-prices"
+             "from" from
+             "to" to
+             "mainnet?" mainnet?
+             "chaos-mode?" chaos-mode?)
```

Comment:
didn't we remove chaos-mode?

Reply:
I have no idea, it is at least in this function

Reply:
@yenda as far as I see it is still in the project

Reply:
i just disabled it, not removed, but dunno if we want to remove all related code completely